include/uart.h

```diff
@@ -173,6 +297,8 @@ struct uart_driver_api {
 
 #endif
 
+#endif /* CONFIG_UART_HIGH_LEVEL_API */
+
 #ifdef CONFIG_UART_LINE_CTRL
 	int (*line_ctrl_set)(struct device *dev, u32_t ctrl, u32_t val);
```

Comment:
Since you add configure API here, I think the old line_ctrl_set/get are no more needed.

Reply:
Configure is used for UART settings, not for manipulating line control signals. I think these functions should stay, they aren't widely used but there are a few use cases.